internal/config/profile.go

```diff
@@ -100,13 +100,12 @@ func newProfile() *profile {
 	if err != nil {
 		log.Fatal(err)
 	}
-	name := "default"
+	name := DefaultProfile
 	np := &profile{
 		name:      &name,
 		configDir: configDir,
 		fs:        afero.NewOsFs(),
 	}
-	np.SetService(CloudService)
```

Comment:
this is set elsewhere -- also it causes the default profile to be created. So if there is no default profile and you run `mongocli config ls`, this would cause `default` to appear